### ISSUE TITLE
error when accelerate launches, doesn't use pre set set settings.

### ISSUE THREAD

**Issue by alexgilseg** [kohya-ss/sd-scripts#440]
My issue: 
The following values were not passed to `accelerate launch` and had defaults used instead:
        `--num_processes` was set to a value of `1`
        `--num_machines` was set to a value of `1`
        `--mixed_precision` was set to a value of `'no'`
        `--dynamo_backend` was set to a value of `'no'`
        
        teh other once are not set but --mixed:_precision is set to bf16, why does it say it is set to no ???

it goes with training and results are useable, I just wonder if they could be better and why this started happening??

Folder whatever: 66 images found
Folder whatever: 1320 steps
max_train_steps = 2112
stop_text_encoder_training = 0
lr_warmup_steps = 211
accelerate launch --num_cpu_threads_per_process=2 "train_network.py" --enable_bucket --pretrained_model_name_or_path="D:/SD/stable-diffusion-webui/models/Stable-diffusion/v1-5-pruned-emaonly.safetensors" --train_data_dir="D:\SD\Bilder\img" --resolution=512,512 --output_dir="D:\SD\Bilder\model" --logging_dir="D:\SD\Bilder\log" --network_alpha="40" --save_model_as=safetensors --network_module=lycoris.kohya --network_args "conv_dim=40" "conv_alpha=40" "algo=lora" --text_encoder_lr=0.5 --unet_lr=1 --network_dim=40 --output_name="whatever" --lr_scheduler_num_cycles="8" --learning_rate="1" --lr_scheduler="cosine" --lr_warmup_steps="211" --train_batch_size="5" --max_train_steps="2112" --save_every_n_epochs="1" --mixed_precision="bf16" --save_precision="bf16" --caption_extension=".txt" --cache_latents --optimizer_type="DAdaptation" --max_data_loader_n_workers="0" --bucket_reso_steps=64 --xformers --bucket_no_upscale  --sample_sampler=euler_a --sample_prompts="D:\SD\Bilder\model\sample\prompt.txt" --sample_every_n_epochs="1"
The following values were not passed to `accelerate launch` and had defaults used instead:
        `--num_processes` was set to a value of `1`
        `--num_machines` was set to a value of `1`
        `--mixed_precision` was set to a value of `'no'`
        `--dynamo_backend` was set to a value of `'no'`
To avoid this warning pass in values for each of the problematic parameters or run `accelerate config`.
prepare tokenizer
Use DreamBooth method.
prepare images.
found directory D:\SD\Bilder\img\whatever contains 66 image files
1320 train images with repeating.
0 reg images.
no regularization images / 正則化画像が見つかりませんでした
[Dataset 0]
  batch_size: 5
  resolution: (512, 512)
  enable_bucket: True
  min_bucket_reso: 256
  max_bucket_reso: 1024
  bucket_reso_steps: 64
  bucket_no_upscale: True

  [Subset 0 of Dataset 0]
    image_dir: "D:\SD\Bilder\img\whatever"
    image_count: 66
    num_repeats: 20
    shuffle_caption: False
    keep_tokens: 0
    caption_dropout_rate: 0.0
    caption_dropout_every_n_epoches: 0
    caption_tag_dropout_rate: 0.0
    color_aug: False
    flip_aug: False
    face_crop_aug_range: None
    random_crop: False
    token_warmup_min: 1,
    token_warmup_step: 0,
    is_reg: False
    class_tokens: ohwx man
    caption_extension: .txt

[Dataset 0]
loading image sizes.
100%|██████████████████████████████████████████████████████████████████████████████████| 66/66 [00:00<00:00, 66.98it/s]
make buckets
min_bucket_reso and max_bucket_reso are ignored if bucket_no_upscale is set, because bucket reso is defined by image size automatically / bucket_no_upscaleが指定された場合は、bucketの解像度は画像サイズから自動計算されるため、min_bucket_resoとmax_bucket_resoは無視されます
number of images (including repeats) / 各bucketの画像枚数（繰り返し回数を含む）
bucket 0: resolution (448, 384), count: 80
bucket 1: resolution (512, 384), count: 240
bucket 2: resolution (512, 448), count: 40
bucket 3: resolution (576, 320), count: 40
bucket 4: resolution (576, 384), count: 500
bucket 5: resolution (576, 448), count: 120
bucket 6: resolution (640, 384), count: 300
mean ar error (without repeats): 0.048183101637731186
prepare accelerator
Using accelerator 0.15.0 or above.
loading model for process 0/1
load StableDiffusion checkpoint
loading u-net: <All keys matched successfully>
loading vae: <All keys matched successfully>
loading text encoder: <All keys matched successfully>
Replace CrossAttention.forward to use xformers
[Dataset 0]
caching latents.
100%|██████████████████████████████████████████████████████████████████████████████████| 66/66 [00:14<00:00,  4.52it/s]
import network module: lycoris.kohya
Using rank adaptation algo: lora
Use Dropout value: 0.0
Create LyCORIS Module
create LyCORIS for Text Encoder: 72 modules.
Create LyCORIS Module
create LyCORIS for U-Net: 278 modules.
enable LyCORIS for text encoder
enable LyCORIS for U-Net
prepare optimizer, data loader etc.
Deprecated: use prepare_optimizer_params(text_encoder_lr, unet_lr, learning_rate) instead of prepare_optimizer_params(text_encoder_lr, unet_lr)
use D-Adaptation Adam optimizer | {}
when multiple learning rates are specified with dadaptation (e.g. for Text Encoder and U-Net), only the first one will take effect / D-Adaptationで複数の学習率を指定した場合（Text EncoderとU-Netなど）、最初の学習率のみが有効になります: lr=0.5
running training / 学習開始
  num train images * repeats / 学習画像の数×繰り返し回数: 1320
  num reg images / 正則化画像の数: 0
  num batches per epoch / 1epochのバッチ数: 264
  num epochs / epoch数: 8
  batch size per device / バッチサイズ: 5
  gradient accumulation steps / 勾配を合計するステップ数 = 1
  total optimization steps / 学習ステップ数: 2112

**Comment by alexgilseg** [comment]
that would be my guess but as you can see in the code "--mixed_precision="bf16"  yet The following values were not passed to accelerate launch and had defaults used instead: --mixed_precision was set to a value of 'no', so for some reason the launch doesn't transfer the info to accelerate?? That's what it looks like to me..

**Comment by laksjdjf** [comment]
I think there is no ralation between the `--mixed_precision` setting in accelerate and the one used in the code. Why not check if it is actually being applied by looking at the learn speed or VRAM usage?
